client/src/components/jobDetails/index.tsx

```diff
@@ -0,0 +1,56 @@
+import { Button } from '@mui/material';
+import { Link } from 'react-router-dom';
+import './index.css';
+
+function jobDetails() {
+  return (
+    <div className="wrapper">
+      <div className="details">
+        <div className="top">
+          <h2>Job Details</h2>
+        </div>
+        <div className="content">
+          <p>
```

Comment:
there's more info related to job such as budget, duration, and category
Also, try to add them as props